main.js

```diff
@@ -15,7 +15,7 @@ var dialog = electron.dialog;
 
 var mainWindow = null;
 
-var debug = false;
+var debug = true;
```

Comment:
Look like you've set again debug to true...

Reply:
a0db5c8 set it to false in here, my bad